gateway-package/src/main/resources/bin/start.sh

```diff
@@ -236,9 +237,19 @@ fi
 #    -Dapiml.service.ipAddress=${ZOWE_IP_ADDRESS:-127.0.0.1} \
 #    -Dapiml.security.auth.jwtKeyAlias=${PKCS11_TOKEN_LABEL:-jwtsecret} \
 
+if [ "${ATTLS_ENABLED}" = "true" ]; then
+  keystore_type=
```

Comment:
Does it really make sense? Why is there not just `ZWE_configs_server_ssl_enabled=false`. What about the ZAAS component? Remember that the keyring could also be used for a different reason than just TLS - to generate a JWT token.

Reply:
Discussed offline. There needs to be a configuration setting to mark a keyring as having a hardware private key set in ICSF.
If AT-TLS is enabled:
- With z/OSMF JWT provider: Ok not to read keyring (z/OSMF signs the JWT)
- With z/OSMF LTPA. Can't work with ICSF currently. Needs private key to sign JWT.
- With SAF provider. Can't work with ICSF currenty. Needs private key to sign JWT.